cpg-core/src/main/java/de/fraunhofer/aisec/cpg/helpers/Benchmark.kt

```diff
@@ -94,7 +105,13 @@ fun printMarkdown(table: List<List<Any>>, headers: List<String>) {
 
     for (row in table) {
         var rowIndex = 0
-        val line = row.joinToString(" | ", "| ", " |") { it.toString().padEnd(lengths[rowIndex++]) }
+        val line =
+            row.joinToString(" | ", "| ", " |") {
+                val str =
+                    if (it is Map<*, *>) "`${it.toString().replace(",", ", ").replace(":", ": ")}`"
```

Comment:
This does not work anymore, as we pass the `TranslationConfiguration` as object and not as HashMap. 
Do we need/want this formatting rule? I just copied it from the original code.

It should do something like this:

| key | value
| - | - |
| conf | {"key":"value","foo":"bar"} |

-> 

| key | value
| - | - |
| conf | `{"key": "value", "foo": "bar"}` |

Reply:
Yes, I know. For me this is ok. I would need to implement some logic in the markdown printer that check for a hash map and puts them in backticks. not sure if it is worth the time. Feel free to do so if you want :)

Reply:
Ok, then we go without the pretty printing of objects for now.
I converted this into a TODO.